please.sh

```diff
@@ -3441,6 +3441,9 @@ build_mingw_w64_git () { # [--only-i686] [--only-x86_64] [--only-aarch64] [--ski
 	--skip-doc-html)
 		sed_makepkg_e="$sed_makepkg_e"' -e s/"\${MINGW_PACKAGE_PREFIX}-\${_realname}-doc-html"//'
 		;;
+        --reset-pkgrel)
+                sed_makepkg_e="$sed_makepkg_e"' -e s/^pkgrel=[0-9][0-9]*$/pkgrel=1/'
```

Comment:
This could probably be `^pkgrel=[1-9][0-9]*$`

Reply:
Well, I used to _sometimes_ use `pkgrel=0` to prepare packages for new versions without actually releasing them yet... Maybe we can leave it at `[0-9][0-9]*`?

Reply:
I'll merge this now, should we agree that the more stringent pattern is needed, I can always change it as a follow-up after v2.43.0 is released.